src/expr/node_manager_template.h

```diff
@@ -172,6 +172,7 @@ class NodeManager
    * would lead to memory leaks. Thus TypeNode are given a DatatypeIndexConstant
    * which is used as an index to retrieve the DType via this call.
    */
+  const DType& getDTypeFor(TypeNode tn) const;
```

Comment:
Can you update the documentation above this function to account for these 2 functions